fastparquet/converted_types.py

```diff
@@ -119,21 +119,13 @@ def from_bytes(d):
     elif ctype == parquet_thrift.ConvertedType.DATE:
         return (data * DAYS_TO_MILLIS).view('datetime64[ns]')
     elif ctype == parquet_thrift.ConvertedType.TIME_MILLIS:
-        out = np.empty(len(data), dtype='int64')
-        time_shift(data, out, 1000000)
-        return out.view('timedelta64[ns]')
```

Comment:
Since this is the only place that this issue arises (other time types are 64bit to start with), you can probably do data = data.astype('int64') and this should *only* be done on win32, to avoid the time and memory of the copy.